testcases/OpTestInbandUsbInterface.py

```diff
@@ -152,13 +82,25 @@ def run_ipmi_cmd_on_host(self, i_cmd):
             raise OpTestError(l_msg)
```

Comment:
Fixed in 7876e1e75db1ef07481d67d5b45ad331803dbf46, which reworks both inband tests.